pages/api/mail.js

```diff
@@ -0,0 +1,45 @@
+// Nodemailer with SES transport, more info here: https://nodemailer.com/transports/ses/
+import { transporter } from "aws/index";
+
+// send some mail
+export default async function handler(req, res) {
+    if (req.method == "POST") {
```

Comment:
nit: is there a util variable like `postRequest` that has "POST" as a constant? If not - we can set it up ourselves for each of the four main HTTP verbs.